src/Apps/Legion/Components/StoreList/StoreList.tsx

```diff
@@ -0,0 +1,135 @@
+    AppPageName
+} from '../../Contexts/NavigationContext/AppNavigationContext.types';
+
+const debugLogging = false;
+const logDebugConsole = getDebugLogger('StoreList', debugLogging);
+
+const getClassNames = classNamesFunction<
+    IStoreListStyleProps,
+    IStoreListStyles
+>();
+
+const columns: IColumn[] = [
+    {
+        key: 'name',
+        name: 'Store',
+        fieldName: 'databaseName',
+        minWidth: 100
+    }
+];
+
+const StoreList: React.FC<IStoreListProps> = (props) => {
+    const { styles } = props;
+
+    // contexts
+    const { AppDataState, AppDataDispatch } = useAppDataContext();
+    const { appNavigationDispatch } = useAppNavigationContext();
+
+    // state
+    const [
+        selectedItem,
+        setSelectedItem
+    ] = useState<ITargetDatabaseConnection | null>(null);
+
+    // hooks
+
+    // data
+    const items: ITargetDatabaseConnection[] = [
+        {
+            databaseName: 'My Estero DB'
+        },
+        {
+            databaseName: 'Database 2'
+        },
+        {
+            databaseName: 'Database 3'
+        }
+    ];
+
+    // callbacks
+    const selection = useRef<Selection>(
+        new Selection({
+            onSelectionChanged: () => {
+                // getSelection returns an array of selected elements, since this is single select first one is always going to be the correct one
+                const selectionValue = selection.current.getSelection()[0];
+                console.log('***Selected', selectionValue);
+                setSelectedItem(
+                    selectionValue ? selectionValue['databaseName'] : null
+                );
+            }
+        })
+    );
+
+    // side effects
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useExtendedTheme()
+    });
+
+    logDebugConsole('debug', 'Render');
+
+    return (
+        <Stack className={classNames.root}>
+            <DetailsList
+                items={items}
+                columns={columns}
+                selectionMode={SelectionMode.single}
+                selection={selection.current}
+                selectionPreservedOnEmptyClick={true}
+            />
+            <DefaultButton
+                text={'next'}
+                onClick={() => {
+                    AppDataDispatch({
+                        type: AppDataContextActionType.SET_TARGET_DATABASE,
+                        payload: {
+                            targetDatabase: {
+                                databaseName: 'My Estero DB'
+                            } // selectedItem
+                        }
+                    });
+                    appNavigationDispatch({
+                        type: AppNavigationContextActionType.NAVIGATE_TO,
+                        payload: {
+                            pageName: AppPageName.FlowPicker
+                        }
+                    });
+                }}
+                // disabled={!AppDataState.targetDatabase}
```

Comment:
This will be uncommented in a later PR right?

Reply:
I need to figure out how to get the selected state to work right. That's sometheing that isn't working right now. But yeah. I'll fix this before I send it.